crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -499,6 +512,51 @@ impl<'a> TimelineEventHandler<'a> {
         self.state.reactions.map.insert(reaction_id, (reaction_sender_data, c.relates_to));
     }
 
+    fn handle_poll_start(&mut self, c: UnstablePollStartEventContent, should_add: bool) {
+        self.add(
+            should_add,
+            TimelineItemContent::Poll(PollState::new(
+                self.ctx.sender.clone(),
+                self.ctx.timestamp,
+                c,
+            )),
+        );
+    }
+
+    fn handle_poll_response(&mut self, c: UnstablePollResponseEventContent) {
+        // TODO(polls): what if the event isn't found? Cache this for later?
+        update_timeline_item!(self, &c.relates_to.event_id.clone(), "vote", |event_item| {
```

Comment:
```suggestion
        update_timeline_item!(self, &c.relates_to.event_id.clone(), "poll response", |event_item| {
```

Reply:
(this is used for the debug message all the way at the end of this file)

Reply:
I was actually wondering for what it was for but forgot to ask. Thanks.